modules/oe_content_call_tenders/src/CallForTendersNodeWrapper.php

```diff
@@ -78,16 +78,16 @@ public function getStatus(): string {
     $now = $this->getNow();
 
     $status = static::STATUS_NOT_AVAILABLE;
-    if (!empty($opening_date)) {
+    if (isset($opening_date)) {
       if ($now < $opening_date) {
         $status = static::STATUS_UPCOMING;
       }
       elseif ($opening_date <= $now && $now < $closing_date) {
```

Comment:
We haven't checked if the closing_date is set here, so if its not we might get an error (although the field is required).

Reply:
True